subset_vds.py

```diff
@@ -63,31 +70,33 @@ def check_output_already_exists(output_format: list[str], infile_name: str) -> N
     files_exist_errors: bool = False
     for format in output_format:
         if format == "vds":
-            if to_path(
+            if vds_path := to_path(
```

Comment:
Gotta check your Walrus, son

```
if path := to_path('gs://cpg-acute-care-main/reanalysis/2024-10-03/config.toml').exists():
    print(path)
    
> True
if (path := to_path('gs://cpg-acute-care-main/reanalysis/2024-10-03/config.toml')).exists():
    print(path)
    
> gs://cpg-acute-care-main/reanalysis/2024-10-03/config.toml
```

If you're using a method on the thing your walrus operator creates, you need to bracket the creation of the object specifically

```if (variable := the thing you create).method():```

Reply:
Never used walri before, so good to learn that.